site/content/blogs/release-v127.en.md

```diff
@@ -87,11 +87,74 @@ Continue with the deployment? (y/N)
+To expose multiple ports through the Application Load Balancer, we will configure additional listener rules through a new `http` field called [`additional_rules`](../docs/manifest/lb-web-service.en.md#http-additional-rules). 
+It is as easy as configuring your `http` field. Let's learn through an example. 
+
+Say we want to expand the basic manifest such that it opens up port 8081 on the main service container, and 8082 on the sidecar container, in addition to the existing `image.port` 8080.
+```yaml
+# Example 1
+name: 'frontend'
+type: 'Load Balanced Web Service'
+ 
+image:
+  build: './frontend/Dockerfile'
+  port: 8080
+  
+http:
+  path: '/'
+  additional_rules:             # The new field "additional_rules",
+    - target_port: 8081        # Optional. Defaults to the `image.port`.
+      path: 'customerdb'
+    - target_port: 8082
+      target_container: nginx   # Optional. Defaults to the main container. 
+      path: 'admin'
+    - target_port: 80
+      path: 'superAdmin'
+
+sidecars:
+  nginx:
+    port: 80
+    image: public.ecr.aws/nginx:latest
+```
+With this manifest, requests to “/” will still be routed to the main container’s port 8080. Requests to "/customerdb" will be route to the main container’s 8081, 
+, "/admin" to “nginx”‘s port 8082 and "/superAdmin" to "nginx"'s port 80. We should note that the third listener rule just defined 'target_port: 80' 
+and Copilot was able to intelligently route traffic from the '/superAdmin' to the sidecar container named nginx.
+
+It is also possible to configure the container port that handles the requests to “/” via our new field under `http` called [`target_port`](../docs/manifest/lb-web-service.en.md#http-target-port)
 
 ### Add multiple port and protocol listeners to your Network Load Balancers
+To expose multiple ports through the Network Load Balancer, we will configure additional listeners through a new `nlb` field called [`additional_listeners`](../docs/manifest/lb-web-service.en.md#nlb-additional-listeners).
+It is as easy as configuring your `nlb` field. Let's learn through an example.
+
+```yaml
+name: 'frontend'
+type: 'Load Balanced Web Service'
+
+image:
+  build: Dockerfile
+
+http: false
+nlb:
+  port: 8080/tcp
+  additional_listeners:
+    - port: 8081/tcp
+    - port: 8082/tcp
+      target_port: 8085               # Optional. Default is set to nlb port of the specific listener 
+      target_container: nginx         # Optional. Default is set to the main container
+
+sidecars:
+  nginx:
+    port: 80
+    image: public.ecr.aws/nginx:latest
+```
+With this manifest, requests to NLB port 8080/tcp will now be routed to the main container’s port 8080. 
+Requests to another NLB port 8081 will be routed to the port 8081 of the main service container. 
+We need to notice here that the default value of the target_port will be the same as that of the corresponding NLB port. 
+The requests to NLB port 8082 will be routed to port 8085 of the sidecar container named nginx.
```

Comment:
what about nginx port 80?

Reply:
Customer can send traffic to port 80 if they want to. But in this case we aren't sending any traffic to port 80. But customer can access this port 80 internally (inside VPC) through SD.